### DISCUSSION TITLE
Small Bug - Folder and contents missing from the install package

### DISCUSSION THREAD

**Discussion by jmourinho** [Leantime/leantime#961] [General] [answered]
When i uploaded a file at the project settings, i get a 404 for the file icon.
In this case this is a PDF:

![Screenshot_2](https://user-images.githubusercontent.com/7406584/190366579-175717ed-f983-4563-9ce9-93993a34f303.png)

![Screenshot_3](https://user-images.githubusercontent.com/7406584/190367086-2f4a9f87-2271-4406-bff3-259a0d098be7.png)

It seems the folder "thumbs" and its supposed contents are not part of the download package.

**Comment by phulstaert** [comment, marked as answer]
Hi @jmourinho,

You -or another user- reported this on our Discord server.
The issue is known and should be resolved in the next version.

If you want to fix it in the mean time, you can download the 2.2.6 release and copy the missing files from the ./public/images folder to your own installation.

If you need help with this, don't hesitate to ask it here or to contact us on Discord.

**Comment by jmourinho** [comment]
Great! Thank you for your help.
